### ISSUE TITLE
[Site Request] Koushoku

### ISSUE THREAD

**Issue by ImperialXeno** [manga-download/hakuneko#5483]
**Name of the website**
Koushoku/ksk

**Website urls (examples below)**
- Site: https://ksk.moe/
- Manga List: https://ksk.moe/browse
- Manga example: https://ksk.moe/archive/9066/get-good-grades-for-me
- Chapter online viewer example: https://ksk.moe/archive/9066/get-good-grades-for-me/2

**Languages**
English
**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
18+/NSFW = Yes

**Comment by kevin01523** [comment]
yeah i hope this gets added

**Comment by NeverSm1le** [comment]
Second this.

**Comment by vex-Cassian** [comment]
svp  y a-t-il  une apk pour télécharger Koushoku ??

**Comment by kevin01523** [comment]
yeah ksk got stable recently and easier to downlad files from it and its static now even if the manga links only consists of generated hash

**Comment by MikeZeDev** [comment]
Interesting API

**Comment by LetrixZ** [comment]
Koushoku was replaced by Anchira

**Comment by MikeZeDev** [comment]
> Koushoku was replaced by Anchira

Yeah, that will change nothing ;D

**Comment by vex-Cassian** [comment]
He didn't work because of copyright issues.

**Comment by MikeZeDev** [comment]
✅ Is added [HaruNeko](https://docs.hakuneko.download/user-manual/)
🚫 Will not be added to HakuNeko
